apps/src/code-studio/components/ExportDialog.jsx

```diff
@@ -381,14 +547,41 @@ class ExportDialog extends React.Component {
     }
   };
 
+  onBackButton = () => {
+    const {screen} = this.state;
+
+    switch (screen) {
+      case 'intro':
+        return;
+      case 'platform':
+        return this.setState({screen: 'intro'});
```

Comment:
Put these on two different lines. It gets confusing when a function that doesn't return anything looks like it's returning something.

Reply:
Same for the others in this area